app/lib/pdf_filler/additional_dependents_pdf.rb

```diff
@@ -9,12 +9,7 @@ def source_pdf_name
 
     def initialize(submission, start_node: 4)
       # For some PDF fields, use values from the database b/c the XML values are truncated or missing.
-      if submission.tax_return.present?
-        @xml_document = SubmissionBuilder::Ty2021::Documents::Irs1040.new(submission).document
-      else
-        # State file does not have a submission.tax_return - so we grab the raw data.
-        @xml_document = Nokogiri::XML(submission.data_source.raw_direct_file_data)
-      end
+      @xml_document = SubmissionBuilder::Ty2021::Documents::Irs1040.new(submission).document
```

Comment:
This is actually a reversion - since statefile does not use this class anymore, I am putting it back to its original state.